frontend/src/components/auth/LoginForm.vue

```diff
@@ -105,27 +105,12 @@
 
 <style lang="scss" scoped>
     .login-form-input {
-        height: 37px;
-        border: 1px solid rgba(18, 39, 55, 0.11);
-        border-radius: 3px;
-
-        ::v-deep {
-            .v-input__control {
-                min-height: 1px;
-            }
-
-            input {
-                min-height: 35px;
-            }
-
-            .v-input__prepend-outer {
-                margin-top: 4px;
-            }
-        }
-
         &.v-input--is-focused {
-            border: 1px solid rgba(60, 87, 222, 0.52);
-            box-shadow: 0px 0px 14px rgba(194, 205, 223, 0.6);
+            ::v-deep {
+                .v-input__slot {
+                    border: 1px solid rgba(60, 87, 222, 0.52);
```

Comment:
you sure that you need this? vutify should already have border specified for inputs

Reply:
Without this property input will not have blue border on focus

Reply:
it seems that this going to be global styles, if so - it's better to define them in root component (where font definitions are placed)